buildpipeline/pipeline.json

```diff
@@ -283,6 +311,75 @@
           }
         },
         {
+          "Name": "DotNet-CoreFx-Trusted-Windows",
+          "Parameters": {
+            "PB_Platform": "arm",
+            "PB_BuildArguments": "-buildArch=arm -Release -portable -- /p:SignType=real /p:RuntimeOS=win10",
+            "PB_BuildTestsArguments": "-buildArch=arm -Release -SkipTests -Outerloop -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
```

Comment:
I think the best way to test this is to do the steps locally. Do a build.cmd -portable and then do a build-tests.cmd without portable. There is a chance that the non-portable assets will end up in the shared framework folder we are testing against.